common/lib/xmodule/xmodule/modulestore/split_mongo/split.py

```diff
@@ -2071,6 +2070,154 @@ def copy(self, user_id, source_course, destination_course, subtree_list=None, bl
             self.update_structure(destination_course, destination_structure)
             self._update_head(destination_course, index_entry, destination_course.branch, destination_structure['_id'])
 
+    @contract(source_keys="list(BlockUsageLocator)", dest_usage=BlockUsageLocator)
+    def inherit_copy(self, source_keys, dest_usage, user_id):
```

Comment:
Using `inherit` here worries me, since there's a risk of conflating it with `InheritanceMixin` and other similar code.